plugin/fzf.vim

```diff
@@ -838,7 +838,7 @@ else
       call setbufline(winbufnr(id), 1, a:opts.border)
       execute 'autocmd BufWipeout * ++once call popup_close('..id..')'
     else
-      execute 'autocmd BufWipeout * ++once call term_sendkeys('..buf..', "exit\<CR>")'
+      execute 'autocmd BufWipeout * ++once '..buf..'bwipeout!'
```

Comment:
Let's change this to

```vim
execute 'autocmd BufWipeout * ++once bwipeout!' buf
```

as it's a bit more concise.

Reply:
Thank you, modified.